DOCS-1068: Add full image view (#1867)
viamrobotics/docs: docs/manage/data/label.md

{{<gif webm_src="/manage/data/label-dog.webm" mp4_src="/manage/data/label-dog.mp4" alt="Add a bounding box around the dog in an image">}}

> If you want to expand the image, click on the expand side menu arrow:

Author: ```suggestion
If you want to expand the image, click on the expand side menu arrow in the corner of the image:
```

> If you want to see the image in bigger, click on the expand side menu arrow:

Reviewer: I think "in bigger" sounds wrong
```suggestion
If you want to see the image bigger, click on the expand side menu arrow.
```
or
```suggestion
If you want to expand the image, click on the expand side menu arrow.
```
maybe both this and the other one should either have a colon or a period?

Author: ```suggestion
If you want to expand the image, click on the expand side menu arrow:
```

{{<gif webm_src="/manage/data/tag-star.webm" mp4_src="/manage/data/tag-star.mp4" alt="Tag image with a star label">}}

> If you want to see the image bigger, click on the expand side menu arrow.

Author: ```suggestion
If you want to expand the image, click on the expand side menu arrow in the corner of the image.
```

> If you want to see the image in bigger, click on the expand side menu arrow.

Reviewer: ```suggestion
If you want to see the image bigger, click on the expand side menu arrow.
```
or
```suggestion
If you want to expand the image, click on the expand side menu arrow.
```
